frontend/src/app/shared/components/autocompleter/user-autocompleter/user-autocompleter.component.ts

```diff
@@ -188,6 +188,9 @@ export class UserAutocompleterComponent extends UntilDestroyedMixin implements O
 
     const filteredURL = addFiltersToPath(this.url, searchFilters);
 
+    // Override pageSize as there can be more that 30 results
+    filteredURL.searchParams.set('pageSize', '-1');
+
     return this
       .halResourceService
       .get<CollectionResource<UserResource>>(filteredURL.toString(), { pageSize: -1 })
```

Comment:
Why is here a `pageSize` defined, too? Doesn't it have any effect? If not, this has to be removed to avoid confusion in future.

Reply:
The `pageSize` in line 193 was added by @ulferts at the same time on a different branch for another reason. This solves the bug. See https://github.com/opf/openproject/pull/11078